src/server/notifications.js

```diff
@@ -16,6 +17,16 @@ async function getOrganizationOwner(organizationId) {
     .limit(1)
     .eqJoin('user_id', r.table('user'))('right')(0)
 }
+
+const getNotificationFields = (organization, campaign) => ({
+  organizationName: organization.name,
+  campaignTitle: campaign.title,
+  todoUrl: `${process.env.BASE_URL}/app/${campaign.organization_id}/todos`
+})
+
+export const ASSIGNMENT_CREATED_SUBJECT = '[{organizationName}] New assignment: {campaignTitle}'
```

Comment:
nit: i get why these are kinda 'big constants', but they're not global, so I think I would vote that they stay normal camel-case variable names.